man/docker-exec.1.md

```diff
@@ -33,6 +34,13 @@ container is unpaused, and then run
 **-i**, **--interactive**=*true*|*false*
    Keep STDIN open even if not attached. The default is *false*.
 
+**--privileged**=*true*|*false*
+   Give extended privileges to the process to run in a running container. The default is *false*.
+
+   By default, the process run by docker exec in a running container
+have the same capabilities of the container. By setting --privileged will give
```

Comment:
Also, could you add that to the `docs/reference/commandline/exec.md` file as well?